apps/judicial-system/web/src/routes/Prosecutor/Indictments/Indictment/IndictmentCount.tsx

```diff
@@ -176,6 +148,7 @@ interface LawsBrokenOption {
   disabled: boolean
 }
 
+// TODO: remove
```

Comment:
Could you please either remove this comment or explain better what needs to happen and when ☺️